util/util.go

```diff
@@ -623,6 +623,18 @@ func RemountVolume(volumeName string) error {
 		return nil
 	}
 
+	// mounting multiple layers with xfs filesystem will get the following error:
+	// 		mount: wrong fs type, bad option, bad superblock on <device path>,
+	// 		       missing codepage or helper program, or other error
+	//
+	// 		       In some cases useful info is found in syslog - try
+	// 		       dmesg | tail or so.
+	// This error is caused by duplicate UUID of the XFS Filesystem
```

Comment:
Put the exact `dmesg` error message here.

Btw, don't need `In some cases ...` part. It's pretty standard.